app/controllers/charge_controller.js

```diff
@@ -147,8 +152,11 @@ module.exports = {
     const startTime = new Date()
     const correlationId = req.headers[CORRELATION_HEADER] || ''
     const connector3dsUrl = paths.generateRoute('connectorCharge.threeDs', {chargeId: charge.id})
-
-    baseClient.post(connector3dsUrl, {data: {pa_response: _.get(req, 'body.PaRes')}, correlationId},
+    const auth3dsPayload = {
+      pa_response: _.get(req, 'body.PaRes'),
+      auth_3ds_result: AUTH_3DS_EPDQ_RESULTS[_.get(req, 'body.providerStatus', '')]
```

Comment:
If it is none of the `AUTH_3DS_EPDQ_RESULTS` the app would crash. Is that expected behaviour?